src/api/component.js

```diff
@@ -19,6 +20,22 @@ import root from 'window-or-global';
 const { HTMLElement } = root;
 const htmlElementPrototype = HTMLElement ? HTMLElement.prototype : {};
 
+
+
+// Prevent double-calling with polyfill.
+
+const $prevName = createSymbol();
+const $prevOldValue = createSymbol();
+const $prevNewValue = createSymbol();
+
+function preventDoubleCalling (elem, name, oldValue, newValue) {
+  return name === elem[$prevName]
```

Comment:
I've created a basic reproduction of what we're doing internally: http://www.webpackbin.com/EyVC4-Agz.

I think we should just continue being defensive, but this does raise some questions. In that bin:

- Chrome CE v1 text content is 1
- Safari CE v1 text content is 1772 (or whatever, call stack exceeded)
- In polyfill text content is 2